blocks/link-list/link-list.js

```diff
@@ -5,40 +5,61 @@ import { isExternalURL } from '../../blocks-helpers/isExternalURL.js';
  * @function
  * @param {string} textContent the desired link text
  * @param {string} url the desired link href
+ * @param {string} [altText] the link title
  * @returns {Element}
  */
 
-const buildLinkListItem = ({ textContent, url }) => {
-  const linkListItem = document.createElement("li");
-  const linkListItemAnchor = document.createElement("a");
-  linkListItemAnchor.className = "link-list-item";
-  linkListItemAnchor.href = url;
+const buildLinkListItem = ({ textContent, url, altText }) => {
+  const item = document.createElement("li");
+  const itemAnchor = document.createElement("a");
+  itemAnchor.className = "link-list-item";
+  itemAnchor.href = url;
+  if (altText) itemAnchor.title = altText;
```

Comment:
```suggestion
  if (altText) itemAnchor.setAttribute("aria-label", altText);
```

if we've done this in other places in the codebase we will need to update

Reply:
I'm updating where we've used title instead of setting the `aria-label` attribute throughout!